edelphi/src/main/java/fi/metatavu/edelphi/query/thesis/AbstractThesisQueryPageHandler.java

```diff
@@ -129,6 +135,11 @@ public void saveAnswers(RequestContext requestContext, QueryPage queryPage, Quer
     }
   }
   
+  @Override
+  public ReportPageCommentProcessor exportComments(QueryPage queryPage, PanelStamp stamp, List<QueryReply> replies) {
+    return new GenericReportPageCommentProcessor(stamp, queryPage, new HashMap<>());
+  }
+  
   @Override
   public List<QueryOption> getDefinedOptions() {
     List<QueryOption> options = new ArrayList<QueryOption>(super.getDefinedOptions());
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Rename "options" which hides the field declared at line 36. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AHiddenFieldCheck)
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Replace the type specification in this constructor call with the diamond operator ("<>"). [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS2293)